src/lib/AnnotationControls.scss

```diff
@@ -1,18 +1,18 @@
 .bp-AnnotationControls-group {
-    padding: 0 4px 0 8px;
+    padding-left: 4px;
     border-left: 1px solid $twos;
 
     .bp-AnnotationControls-regionBtn {
-        width: $controls-button-width;
-        height: $controls-button-width;
-        border-radius: 4px;
-
         svg {
             fill: $white;
         }
 
         &.is-active {
+            width: 34px;
```

Comment:
Seems like these measurements should remain the same whether `is-active` is applied or not.